_posts/en/newsletters/2024-01-10-newsletter.md

```diff
@@ -0,0 +1,288 @@
+---
+title: 'Bitcoin Optech Newsletter #284'
+permalink: /en/newsletters/2024/01/10/
+name: 2024-01-10-newsletter
+slug: 2024-01-10-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Discussion about proposed v3 transaction relay policy:**
+  Antoine Poinsot [posted][poinsot v3] to Delving Bitcoin to
+  foster`discussion about the proposals for [v3 transaction relay
+  policy][topic v3 transaction relay] and [ephemeral anchors][topic
+  ephemeral anchors].  The thread appears to have been motivated by
+  Peter Todd [posting][todd v3] a critique of v3 relay policy on his
+  blog.  We've arbitrarily divided discussion in to several parts:
+
+  - **Frequent use of exogenous fees may risk mining decentralization:**
+    an ideal version of the Bitcoin protocol would reward each miner
+    proportionately to their hashrate.  The implicit fees paid in
+    transactions preserve that property: a miner with 10% of total
+    hashrate has a 10% chance of capturing a next-block fee, whereas a
+    miner with 1% of hashrate has a 1% chance.  Fees paid outside of
+    transactions and directly to miners, called [out-of-band fees][topic out-of-band fees],
+    violate that property: a system that pays miners who together control
+    over 55% of hashrate has a 99% chance of getting a transaction
+    confirmed within 6 blocks<!-- 1 - (1 - 0.55)**6 -->, likely resulting
+    in little effort being made to pay small miners of 1% or less
+    hashrate.  If small miners are paid less than large miners, mining
```

Comment:
Perhaps:
```suggestion
    hashrate.  If small miners are paid less than large miners in proportion to the blocks they find, mining
```
or
```suggestion
    hashrate.  If smaller miners cannot profit without access to out-of-band fees, mining
```

Since it's expected that less hashrate => less revenue, but it's problematic when out-of-band fees are significant.